includes/admin/class-wcs-admin-meta-boxes.php

```diff
@@ -247,21 +247,62 @@ public static function process_renewal_action_request( $subscription ) {
 	/**
 	 * Handles the action request to create a pending renewal order.
 	 *
-	 * @param array $subscription
-	 * @since 1.0.0 - Migrated from WooCommerce Subscriptions v2.0
+	 * @param WC_Subscription $subscription
 	 */
 	public static function create_pending_renewal_action_request( $subscription ) {
-		$subscription->add_order_note( __( 'Create pending renewal order requested by admin action.', 'woocommerce-subscriptions' ), false, true );
```

Comment:
Just in case it gets missed, we should revert this as per the discussion here: https://github.com/Automattic/woocommerce-subscriptions-core/pull/768#discussion_r1924734925

Reply:
Restored in https://github.com/Automattic/woocommerce-subscriptions-core/pull/768/commits/2deaa0fdea0bb112d6ea198e95f518485ce830e0.